Assets/Scripts/ProjectileController.cs

```diff
@@ -12,53 +12,60 @@ public class ProjectileController : MonoBehaviour
     private float projDeviation = 1f;
 
     public enum TargetName { Player, Enemy };
-
-    // These variables need to be set on creation of the instance
     public TargetName tName; // "Player" to damage players, "Enemy" to damage enemies
-    public Transform target; // Transform of gameObject to target
 
-    void Start()
+    private Transform target;
+    private Collider2D senderColl;
+
+    // Sets up the variables needed to be set on creation of the instance
+    public void SetUpProjectile(Transform t, Collider2D sC)
```

Comment:
I think it's better to use `Transform target, Collider2D senderColl` for the parameter list. Then, it's easier to tell what the parameters are for in an IDE tooltip for a call to this method.
Generally, it's good to write informative parameter names for method documentation (like with Javadocs or XML documentation in C#) or creating interfaces from classes, but it's unlikely those things will matter in this project.